src/components/PostList/PostList.jsx

```diff
@@ -2,0 +1,9 @@
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} key={post.id} />
```

Comment:
The `key` prop is correctly set to `post.id`. Ensure that each post object has a unique `id` to prevent rendering issues.